ambari/administration/user-group.rst

```diff
@@ -0,0 +1,182 @@
+Managing Users and Groups
+=========================
+
+An "Ambari Admin" can create and manage users and groups available to Ambari. An Ambari Admin can also import user and group information into Ambari from external LDAP systems. This section describes the specific tasks you perform when managing users and groups in Ambari.
+
+* :ref:`Local and LDAP User Types<ambari-user-group-types>`
```

Comment:
Index is not coherent with section structure. For example _User and Groups Overview_ (1st level) is not listed while _Ambari Admin Privileges_ (2nd level) is. Using only 1st level section is ok.